@xen-orchestra/lite/src/components/pool/dashboard/AlarmRow.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <tr v-if="isMobile" class="mobile" @click="handleCollapse">
+    <td v-if="isMobile" class="description" colspan="5">
+      <div v-tooltip class="ellipsis">
+        <ArrowCollapseButton :is-collapsed="!isDescriptionCollapsedRef" />
+        {{ $t(`alarm-type.${alarm.type}`, { n: alarm.triggerLevel * 100 }) }}
+      </div>
+    </td>
+  </tr>
+
+  <tr v-if="!isDescriptionCollapsedRef && isMobile" class="collapsible-description">
+    <td class="collapsible-description__content" colspan="5">
+      {{ tempDescription }}
+    </td>
+  </tr>
+  <tr v-if="isMobile" class="alarm-details" colspan="5">
```

Comment:
```suggestion
  <tr v-if="isMobile" class="alarm-details">
```

`colspan` attribute does not exist on `tr`